extensions/Worker.Extensions.Storage.Blobs/src/BlobInputAttribute.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.Azure.Functions.Worker
 {
-    [AllowConverterFallback(false)]
+    [AllowConverterFallback(true)]
```

Comment:
Similar to [BlobTriggerAttribute](https://github.com/Azure/azure-functions-dotnet-worker/blob/bcaf5e73f735db2137d3253e22241353407ac3bc/extensions/Worker.Extensions.Storage.Blobs/src/BlobTriggerAttribute.cs#L9)